Source/Scene/Implicit3DTileContent.js

```diff
@@ -137,6 +138,15 @@ Object.defineProperties(Implicit3DTileContent.prototype, {
       return undefined;
     },
   },
+
+  metadataGroup: {
```

Comment:
Also:

> How should group metadata interact with external tilesets

Looks like the code goes with the simpler approach of only assigning groups to `Tileset3DTileContent` and not contents in the external tileset, which is probably for the best because it avoids nested groups. Just want to confirm that that's the behavior.

Reply:
>Does this need to be tested further or is it covered by existing unit tests?

Ah, I think I've tested all the parts of this behavior separately, but no overall test. I'm working on a new sample to try loading a tileset with implicit + multi-content + group metadata

>Just want to confirm that that's the behavior.

Correct, I'm just keeping this simple, as propagating groups down the inner tree sounds like it could get messy fast.